src/FSharp.AWS.DynamoDB/TableContext.fs

```diff
@@ -439,17 +423,15 @@ type TableContext<'TRecord> internal
     ///     Asynchronously checks whether item of supplied key exists in table.
     /// </summary>
     /// <param name="key">Key to be checked.</param>
-    /// <param name="collector">Function to receive request metrics.</param>
-    member __.ContainsKeyAsync(key : TableKey, ?collector) : Async<bool> = async {
+    member __.ContainsKeyAsync(key : TableKey) : Async<bool> = async {
         let kav = template.ToAttributeValues(key)
         let request = GetItemRequest(tableName, kav)
         request.ExpressionAttributeNames.Add("#HKEY", template.PrimaryKey.HashKey.AttributeName)
         request.ProjectionExpression <- "#HKEY"
-        let returnConsumedCapacity, maybeReport = metricsOptions collector
         request.ReturnConsumedCapacity <- returnConsumedCapacity
         let! ct = Async.CancellationToken
         let! response = client.GetItemAsync(request, ct) |> Async.AwaitTaskCorrect
-        match maybeReport with None -> () | Some r -> r GetItem [ response.ConsumedCapacity ] 0
+        maybeReport |> Option.iter (fun r -> r GetItem [ response.ConsumedCapacity ] 0)
```

Comment:
While I know we didnt yield the full item, we still paid to read it so I'd argue for a `1` here? I do agree it's subjective though

Reply:
Yes, that’s fair enough. Also no test for this, and I've added a 1 to `DeleteItem` also so it’s consistent with the Batch call